x-pack/legacy/plugins/apm/public/utils/flattenObject.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { compact, isObject } from 'lodash';
+
+export type FlattenItems = Array<{ key: string; value: unknown }>;
+
+export const flattenObject = (
+  item: Record<string, any>,
```

Comment:
```suggestion
  item: Record<string, unknown>,
```

Reply:
If I change it to `unknown` I got this message `Argument of type 'unknown' is not assignable to parameter of type 'Record<string, unknown>'`, when I recursively call flattenObject here `return acc.concat(flattenObject(item[key], currentKey));`